app/src/common/utils/getRolesMap.js

```diff
@@ -0,0 +1,12 @@
+import { PROJECT_ROLES } from 'common/constants/projectRoles';
+
+export function getRolesMap() {
+  const roles = [];
+  PROJECT_ROLES.forEach((role) => {
+    roles.push({
+      value: role,
+      label: role.split('_').join(' '),
+    });
+  });
+  return roles;
+}
```

Comment:
I'd suggest use `map` method here
```
export function getRolesMap() {
  return PROJECT_ROLES.map(role =>  ({
    value: role,
    label: role.split('_').join(' ')
  });
}
```

Reply:
As far as we using a list of constants as a source, I assume that we can calculate a result of this function only once and store it as a constant